### DISCUSSION TITLE
Slider bodies outlines are thinner on lazer

### DISCUSSION THREAD

**Discussion by Theighlin** [ppy/osu#25057] [Q&A] [answered]
This one i noticed in my ongoing investigations between lazer and stable differences. (i'm hoping this is a one-liner change)

| Stable | Lazer |
|--------|--------|
| ![image](https://github.com/ppy/osu/assets/53081750/d685c297-b934-46de-b2a0-5f4a850e8da4) | ![image](https://github.com/ppy/osu/assets/53081750/2f352908-3b32-4fc8-bbe7-6a6427386914) |
| ![image](https://github.com/ppy/osu/assets/53081750/7f9d16d9-2659-49dd-84df-cc74c2480f82) | ![image](https://github.com/ppy/osu/assets/53081750/e8a62edc-6a00-4305-b86b-32ed934a74cb) |
| ![image](https://github.com/ppy/osu/assets/53081750/333ff8e9-fec1-4efc-a043-c0368c8055aa) | ![image](https://github.com/ppy/osu/assets/53081750/9751733d-b07f-42d2-be43-c7cc809f523a) |

Honestly, i don't know how far i should be taking this. I think lazer and stable might never be able to look the same, and that's fine.
I don't want to come off as annoying in doing this, so tell me if i should stop or do it when you have less important things going on.

**Comment by peppy** [comment]
Could you please provide screenshots using default skins if the skin is not relevant? It would be helpful in better isolating the issue.

It's also important to see if the thinner lines match the default skins better, in which case they may be preferrable.

**Reply by Theighlin** [reply]
> Could you please provide screenshots using default skins if the skin is not relevant?

Noted, edited OP.

> It's also important to see if the thinner lines match the default skins better

Subjective, thicker slider body lines match the border of the hitcircle in the classic skin, which is better imo (can't say for triangles and argon).
I know these are not the ground-breaking differences that prevent players from switching, far from it. I just think there is value in pointing them out, since basically everything else is trying to match stable, unless there is a good reason not to.

**Reply by peppy** [reply]
I've gone through the stable code to try and find the correct numbers for this, but it's a bit of a mess and I'm really not sure.

Might be easiest to just adjust based on a visual comparison.

For anyone attempting this, we should also probably remove [SliderBorderSize](https://github.com/ppy/osu/blob/82ff142b1b379de24550f314241ddbc82965d4c4/osu.Game.Rulesets.Osu/Skinning/OsuSkinConfiguration.cs#L8) as being configurable, since it isn't on stable and isn't really implemented correctly anyway.

**Comment by frenzibyte** [comment, marked as answer]
https://github.com/ppy/osu/issues/26000
